scripts/assets/functions_to_test.js

```diff
@@ -2,34 +2,34 @@ var functions = require("firebase-functions");
 var admin = require("firebase-admin");
 admin.initializeApp(functions.config().firebase);
 
-exports.dbAction = functions.database.ref("/input/{uuid}").onWrite(function(event) {
-  console.log("Received event:", event);
-  return event.data.ref.root.child("output/" + event.params.uuid).set(event.data.val());
+exports.dbAction = functions.database.ref("/input/{uuid}").onCreate(function(snap, context) {
```

Comment:
just curious about why the switch from onWrite to onCreate for this

Reply:
So the function receives a Snapshot, instead of a Change object. (We didn't need the "before" and "after" that Change provides.)